src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -1,9 +1,13 @@
 package edu.kis.vh.nursery;
 
-public class FIFORhymer extends defaultCountingOutRhymer {
+public class FIFORhymer extends DefaultCountingOutRhymer {
 
-	public defaultCountingOutRhymer temp = new defaultCountingOutRhymer();
-	
+	public final DefaultCountingOutRhymer temp = new DefaultCountingOutRhymer();
+
+	/***
+	 * metoda zwraca ostatnią liczbe z tablicy numbers
+	 * @return zwracana wartość
+	 */
 	@Override
 	public int countOut() {
 		while (!callCheck())
```

Comment:
Indentation below is not fixed.
Such empty lines below could be removed (for readability).